app_legacy/Helpers/database/game.php

```diff
@@ -817,13 +817,18 @@ function submitNewGameTitleJSON($user, $md5, $gameIDin, $titleIn, $consoleID, $d
     return $retVal;
 }
 
+/**
+ * Slash-escape special title characters (`\`, `'` and `/`)
+ */
 function sanitizeTitle(string $titleIn): string
 {
-    // Remove single quotes, replace with double quotes:
-    $title = str_replace("'", "''", $titleIn);
-    $title = str_replace("/", "-", $title);
+    $title = $titleIn;
+    $tokens = ["\\", "'", "/"];
+    foreach ($tokens as $token) {
+        $title = str_replace($token, "\\$token", $title);
+    }
 
-    return str_replace("\\", "-", $title);
+    return $title;
```

Comment:
`sanitizeTitle` should be discarded in favor of just passing `$gameTitle` to `sanitize_sql_inputs`:
```
function getGameIDFromTitle($gameTitle, $consoleID): int
{
    sanitize_sql_inputs($consoleID, $gameTitle);
    settype($consoleID, 'integer');
```

It looks like it was added in #468. I'm not sure why `sanitize_sql_inputs` was not used there.

Reply:
Note that it's also called from `createNewGame`, so a similar change would need to be made there.

Reply:
Well, there goes my solution... at least it's nice to be able to remove extra code.

Should I just delete `sanitizeTitle` then? It's not used anywhere else besides those two functions.